ultet/src/main/scala/za/co/absa/ultet/Ultet.scala

```diff
@@ -26,6 +27,23 @@ import java.sql.{Connection, DriverManager, ResultSet}
 import scala.util.{Failure, Success, Try}
 
 object Ultet {
+  private val logger = Logger(getClass.getName)
+
+  private def runEntries(dbProperties: DBProperties, dbConnection: String, entries: Seq[SQLEntry]): Unit = {
+    val connection: Connection = DriverManager.getConnection(dbConnection, dbProperties.user, dbProperties.password)
```

Comment:
Would make it implicit at `Ultet` object level and set `autocommit` to true (I don't know what it is set to by default).